test/http2c.erl

```diff
@@ -3,7 +3,7 @@
 %% PLEASE ONLY USE FOR TESTING
 %%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-
+-compile({parse_transform, lager_transform}).
```

Comment:
for some reason, dialyzer wasn't finding lager, but only on files in the `./test` directory.